manifests/enterprise/dashboard/service.pp

```diff
@@ -0,0 +1,38 @@
+# = Class: sensu::enterprise::dashboard
+#
+# Installs the Sensu Enterprise Dashboard
+class sensu::enterprise::dashboard::service (
+  $hasrestart = true,
+) {
+
+  validate_bool($hasrestart)
+
+  if $caller_module_name != $module_name {
+    fail("Use of private class ${name} by ${caller_module_name}")
+  }
+
+  if $::sensu::manage_services {
+    case $::sensu::enterprise_dashboard {
+      true: {
+        $ensure = 'running'
+        $enable = true
+      }
+      default: {
+        $ensure = 'stopped'
+        $enable = false
+      }
+    }
+
+    service { 'sensu-enterprise-dashboard':
+      ensure     => $ensure,
```

Comment:
If $::sensu::manage_services is false this will always stop the enterprise dashboard instead of just doing nothing - is that the intended behavior?